tsa/tsacmd/command.go

```diff
@@ -39,7 +43,10 @@ type TSACommand struct {
 
 	ATCURLs []flag.URL `long:"atc-url" required:"true" description:"ATC API endpoints to which workers will be registered."`
 
-	SessionSigningKey *flag.PrivateKey `long:"session-signing-key" required:"true" description:"Path to private key to use when signing tokens in reqests to the ATC during registration."`
+	ClientID     string   `long:"client-id" required:"true" description:"Client used to fetch a token from the auth server"`
+	ClientSecret string   `long:"client-secret" required:"true" description:"Client used to fetch a token from the auth server"`
+	TokenURL     flag.URL `long:"token-url" required:"true" description:"Token endpoint of the auth server"`
+	Scopes       []string `long:"scope" description:"Scopes to request from the auth server"`
```

Comment:
This is not required as the TSA client only uses the client creds grant type.

Reply:
The client credentials grant type still accepts scopes (in the spec), it's just in our case they aren't required by dex. I'd still like keep these here, in case our implementation of the client_credentials grant type in dex needs to change (it hasn't been merged upstream yet).